This patch adds support for el_GR@euro.  It also updates el_GR,
ur_IN and wal_ET to i18n:2012.
These changes are a *subset* of this upstream patch and, therefore, should
not be pushed upstream.

commit 1a06eee86989d7058d317f3643355f8ec5e87efb
Author: Mike Frysinger <vapier@gentoo.org>
Date:   Wed Apr 13 11:22:00 2016 -0400

    localedata: LC_IDENTIFICATION.category: set to ISO 30112 2014 standard

    The ISO 30112 standard defines the valid values for the category
    keyword as only a few options:
        posix:1993
        i18n:2004
        i18n:2012

    The vast majority of locales had changed the "i18n" string to the
    name of its own locale (e.g. "ak_GH:2013") as well as tweaking the
    date (presumably thinking it should be the date of submission).

    Convert all of them to "i18n:2012" for consistency.  A follow up
    change will update localedef to actually check/validate the field.


diff -Nrup a/localedata/locales/el_GR b/localedata/locales/el_GR
--- a/localedata/locales/el_GR	2012-12-24 22:02:13.000000000 -0500
+++ b/localedata/locales/el_GR	2018-06-10 10:23:23.156283677 -0400
@@ -1,6 +1,13 @@
 comment_char %
-escape_char  /
-%
+escape_char /
+
+% This file is part of the GNU C Library and contains locale data.
+% The Free Software Foundation does not claim any copyright interest
+% in the locale data contained in this file.  The foregoing does not
+% affect the license of the GNU C Library as a whole.  It does not
+% exempt you from the conditions of the license if your use would
+% otherwise be governed by that license.
+
 % Greek Language Locale for Greece
 % Source: RAP
 % Address: Sankt Jo//rgens Alle 8
@@ -15,8 +22,6 @@ escape_char  /
 % Date: 1996-10-15
 % Users: general
 % Charset: ISO-8859-7
-% Distribution and use is free, also
-% for commercial purposes.
 
 LC_IDENTIFICATION
 title      "Greek locale for Greece"
@@ -30,19 +35,19 @@ language   "Greek"
 territory  "Greece"
 revision   "1.0"
 date       "2000-06-29"
-%
-category  "el_GR:2000";LC_IDENTIFICATION
-category  "el_GR:2000";LC_CTYPE
-category  "el_GR:2000";LC_COLLATE
-category  "el_GR:2000";LC_TIME
-category  "el_GR:2000";LC_NUMERIC
-category  "el_GR:2000";LC_MONETARY
-category  "el_GR:2000";LC_MESSAGES
-category  "el_GR:2000";LC_PAPER
-category  "el_GR:2000";LC_NAME
-category  "el_GR:2000";LC_ADDRESS
-category  "el_GR:2000";LC_TELEPHONE
 
+category "i18n:2012";LC_IDENTIFICATION
+category "i18n:2012";LC_CTYPE
+category "i18n:2012";LC_COLLATE
+category "i18n:2012";LC_TIME
+category "i18n:2012";LC_NUMERIC
+category "i18n:2012";LC_MONETARY
+category "i18n:2012";LC_MESSAGES
+category "i18n:2012";LC_PAPER
+category "i18n:2012";LC_NAME
+category "i18n:2012";LC_ADDRESS
+category "i18n:2012";LC_TELEPHONE
+category "i18n:2012";LC_MEASUREMENT
 END LC_IDENTIFICATION
 
 LC_COLLATE
@@ -58,8 +63,10 @@ translit_end
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U03BD><U039D><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U03BF><U039F><U006E><U004E><U005D><U002E><U002A>"
+yesexpr "<U005E><U005B><U002B><U0031><U03BD><U039D><U0079><U0059><U005D>"
+noexpr  "<U005E><U005B><U002D><U0030><U03BF><U039F><U006E><U004E><U005D>"
+yesstr  "<U03BD><U03B1><U03B9>"
+nostr   "<U03CC><U03C7><U03B9>"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -139,15 +146,12 @@ t_fmt_ampm "<U0025><U0049><U003A><U0025>
 %
 % Appropriate date representation (date(1))   "%a %d %b %Y %r %Z"
 date_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0072><U0020><U0025><U005A>"
-first_workday 2
+week 7;19971130;4
 first_weekday 2
 END LC_TIME
 
 LC_PAPER
-% FIXME
-height   297
-% FIXME
-width    210
+copy "i18n"
 END LC_PAPER
 
 LC_TELEPHONE
@@ -158,8 +162,7 @@ int_select     "<U0030><U0030>"
 END LC_TELEPHONE
 
 LC_MEASUREMENT
-% FIXME
-measurement    1
+copy "i18n"
 END LC_MEASUREMENT
 
 LC_NAME
@@ -173,7 +176,18 @@ postal_fmt    "<U0025><U0066><U0025><U00
 <U0020><U0025><U0068><U0020><U0025><U0065><U0020><U0025><U0072><U0025>/
 <U004E><U0025><U007A><U0020><U0025><U0054><U0025>/
 <U004E><U0025><U0063><U0025><U004E>"
+country_name "<U0395><U03BB><U03BB><U03AC><U03B4><U03B1>"
 country_ab2 "<U0047><U0052>"
 country_ab3 "<U0047><U0052><U0043>"
 country_num 300
+% GR
+country_car    "<U0047><U0052>"
+% ελληνικά
+lang_name    "<U03B5><U03BB><U03BB><U03B7><U03BD><U03B9><U03BA><U03AC>"
+% el
+lang_ab      "<U0065><U006C>"
+% ell
+lang_term    "<U0065><U006C><U006C>"
+% gre
+lang_lib    "<U0067><U0072><U0065>"
 END LC_ADDRESS
diff -Nrup a/localedata/locales/el_GR@euro b/localedata/locales/el_GR@euro
--- a/localedata/locales/el_GR@euro	2012-12-24 22:02:13.000000000 -0500
+++ b/localedata/locales/el_GR@euro	2018-06-08 14:34:49.809871475 -0400
@@ -1,5 +1,12 @@
-escape_char /
 comment_char %
+escape_char /
+
+% This file is part of the GNU C Library and contains locale data.
+% The Free Software Foundation does not claim any copyright interest
+% in the locale data contained in this file.  The foregoing does not
+% affect the license of the GNU C Library as a whole.  It does not
+% exempt you from the conditions of the license if your use would
+% otherwise be governed by that license.
 
 LC_IDENTIFICATION
 title      "Greek locale for Greece with Euro"
@@ -13,19 +20,19 @@ language   "Greek"
 territory  "Greece"
 revision   "1.0"
 date       "2000-06-24"
-%
-category  "el_GR@euro:2000";LC_IDENTIFICATION
-category  "el_GR@euro:2000";LC_CTYPE
-category  "el_GR@euro:2000";LC_COLLATE
-category  "el_GR@euro:2000";LC_TIME
-category  "el_GR@euro:2000";LC_NUMERIC
-category  "el_GR@euro:2000";LC_MONETARY
-category  "el_GR@euro:2000";LC_MESSAGES
-category  "el_GR@euro:2000";LC_PAPER
-category  "el_GR@euro:2000";LC_NAME
-category  "el_GR@euro:2000";LC_ADDRESS
-category  "el_GR@euro:2000";LC_TELEPHONE
 
+category "i18n:2012";LC_IDENTIFICATION
+category "i18n:2012";LC_CTYPE
+category "i18n:2012";LC_COLLATE
+category "i18n:2012";LC_TIME
+category "i18n:2012";LC_NUMERIC
+category "i18n:2012";LC_MONETARY
+category "i18n:2012";LC_MESSAGES
+category "i18n:2012";LC_PAPER
+category "i18n:2012";LC_NAME
+category "i18n:2012";LC_ADDRESS
+category "i18n:2012";LC_TELEPHONE
+category "i18n:2012";LC_MEASUREMENT
 END LC_IDENTIFICATION
 
 LC_CTYPE
@@ -63,7 +70,6 @@ LC_NAME
 copy "el_GR"
 END LC_NAME
 
-
 LC_ADDRESS
 copy "el_GR"
 END LC_ADDRESS
diff -Nrup a/localedata/locales/ur_IN b/localedata/locales/ur_IN
--- a/localedata/locales/ur_IN	2018-06-08 14:15:14.876886162 -0400
+++ b/localedata/locales/ur_IN	2018-06-08 15:20:51.893461943 -0400
@@ -1,5 +1,13 @@
-comment_char    %
-escape_char     /
+comment_char %
+escape_char /
+
+% This file is part of the GNU C Library and contains locale data.
+% The Free Software Foundation does not claim any copyright interest
+% in the locale data contained in this file.  The foregoing does not
+% affect the license of the GNU C Library as a whole.  It does not
+% exempt you from the conditions of the license if your use would
+% otherwise be governed by that license.
+
 % Urdu language locale for India.
 % Contributed by Pravin Satpute <psatpute@redhat.com> and
 % Mrs. Nasreen Khan
@@ -17,20 +25,19 @@ language    "Urdu"
 territory   "India"
 revision    "1.0"
 date        "2009,June,09"
-%
-category  "ur_IN:2009";LC_IDENTIFICATION
-category  "ur_IN:2009";LC_CTYPE
-category  "ur_IN:2009";LC_COLLATE
-category  "ur_IN:2009";LC_TIME
-category  "ur_IN:2009";LC_NUMERIC
-category  "ur_IN:2009";LC_MONETARY
-category  "ur_IN:2009";LC_MESSAGES
-category  "ur_IN:2009";LC_PAPER
-category  "ur_IN:2009";LC_NAME
-category  "ur_IN:2009";LC_ADDRESS
-category  "ur_IN:2009";LC_TELEPHONE
-
 
+category "i18n:2012";LC_IDENTIFICATION
+category "i18n:2012";LC_CTYPE
+category "i18n:2012";LC_COLLATE
+category "i18n:2012";LC_TIME
+category "i18n:2012";LC_NUMERIC
+category "i18n:2012";LC_MONETARY
+category "i18n:2012";LC_MESSAGES
+category "i18n:2012";LC_PAPER
+category "i18n:2012";LC_NAME
+category "i18n:2012";LC_ADDRESS
+category "i18n:2012";LC_TELEPHONE
+category "i18n:2012";LC_MEASUREMENT
 END LC_IDENTIFICATION
 
 
@@ -132,20 +139,15 @@ t_fmt       "<U0025><U0049><U003A><U0025
 % Appropriate 12 h time representation (%r)
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053>/
 <U0020><U0025><U0070><U0020><U0025><U005A>"
-%
-% Appropriate date representation (date(1))   "%a %b %e %H:%M:%S %Z %Y"
-date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
-<U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
-<U0025><U005A><U0020><U0025><U0059>"
+week 7;19971130;1
 END LC_TIME
 
 
 LC_MESSAGES
-yesexpr     "<U005E><U005B><U06C1><U0079><U0059><U005D><U002E><U002A>"
-noexpr      "<U005E><U005B><U0646><U006E><U004E><U005D><U002E><U002A>"
-
-yesstr      "<U06C1><U0627><U06BA>"
-nostr       "<U0646><U06C1><U06CC><U06BA>"
+yesexpr "<U005E><U005B><U002B><U0031><U06C1><U0079><U0059><U005D>"
+noexpr  "<U005E><U005B><U002D><U0030><U0646><U006E><U004E><U005D>"
+yesstr  "<U06C1><U0627><U06BA>"
+nostr   "<U0646><U06C1><U06CC><U06BA>"
 END LC_MESSAGES
 
 
@@ -170,7 +172,20 @@ END LC_NAME
 
 
 LC_ADDRESS
-copy "hi_IN"
+postal_fmt  "<U0025><U007A><U0025><U0063><U0025><U0054><U0025><U0073>/
+<U0025><U0062><U0025><U0065><U0025><U0072>"
+country_name "<U0628><U06BE><U0627><U0631><U062A>"
+country_ab2 "<U0049><U004E>"
+country_ab3 "<U0049><U004E><U0044>"
+country_num 356
+% IND
+country_car    "<U0049><U004E><U0044>"
+% ur
+lang_ab      "<U0075><U0072>"
+% urd
+lang_term    "<U0075><U0072><U0064>"
+% urd
+lang_lib    "<U0075><U0072><U0064>"
 END LC_ADDRESS
 
 
diff -Nrup a/localedata/locales/wal_ET b/localedata/locales/wal_ET
--- a/localedata/locales/wal_ET	2012-12-24 22:02:13.000000000 -0500
+++ b/localedata/locales/wal_ET	2018-06-08 14:34:16.440637977 -0400
@@ -1,10 +1,16 @@
-comment_char    %
-escape_char     /
+comment_char %
+escape_char /
 
+% This file is part of the GNU C Library and contains locale data.
+% The Free Software Foundation does not claim any copyright interest
+% in the locale data contained in this file.  The foregoing does not
+% affect the license of the GNU C Library as a whole.  It does not
+% exempt you from the conditions of the license if your use would
+% otherwise be governed by that license.
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %
-% Sidama language locale for Ethiopia.
+% Walaita language locale for Ethiopia.
 %
 % Charset: UTF-8
 %
@@ -27,24 +33,23 @@ contact    ""
 email      "locales@geez.org"
 tel        ""
 fax        ""
-language   "wal"
-territory  "ET"
+language   "Wolaytta"
+territory  "Ethiopia"
 revision   "0.10"
 date       "2004-04-25"
-%
-category  "i18n:2000";LC_IDENTIFICATION
-category  "i18n:2000";LC_COLLATE
-category  "i18n:2000";LC_CTYPE
-category  "i18n:2000";LC_MEASUREMENT
-category  "i18n:2000";LC_MONETARY
-category  "posix:1993";LC_NUMERIC
-category  "i18n:2000";LC_PAPER
-category  "i18n:2000";LC_TELEPHONE
-category  "i18n:2000";LC_ADDRESS
-category  "i18n:2000";LC_MESSAGES
-category  "i18n:2000";LC_NAME
-category  "i18n:2000";LC_TIME
-%
+
+category "i18n:2012";LC_IDENTIFICATION
+category "i18n:2012";LC_COLLATE
+category "i18n:2012";LC_CTYPE
+category "i18n:2012";LC_MEASUREMENT
+category "i18n:2012";LC_MONETARY
+category "i18n:2012";LC_NUMERIC
+category "i18n:2012";LC_PAPER
+category "i18n:2012";LC_TELEPHONE
+category "i18n:2012";LC_ADDRESS
+category "i18n:2012";LC_MESSAGES
+category "i18n:2012";LC_NAME
+category "i18n:2012";LC_TIME
 END LC_IDENTIFICATION
 
 
@@ -65,7 +70,7 @@ END LC_CTYPE
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %
-%  Ethiopia Specifc and Shared Data:
+%  Ethiopia Specific and Shared Data:
 %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
@@ -92,7 +97,7 @@ END LC_TELEPHONE
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %
-%  Sidama Specific Data:
+%  Walaita Specific Data:
 %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
@@ -107,14 +112,12 @@ country_post  "<U0045><U0054><U0048>"
 country_ab2   "<U0045><U0054>"
 country_ab3   "<U0045><U0054><U0048>"
 country_num   231   % 210 found in at least one ISO 3166 doc
-% country_car   unknown
+% ETH
+country_car    "<U0045><U0054><U0048>"
 % country_isbn  unknown, Need ISO 2108
 lang_name     "<U12C8><U120B><U12ED><U1273><U1271>"
 lang_term     "<U0077><U0061><U006C>"
 lang_lib      "<U0077><U0061><U006C>"
-
-
-%
 END LC_ADDRESS
 
 
@@ -232,5 +235,5 @@ date_fmt    "<U0025><U0041><U1363><U0020
 <U0020><U130B><U120B><U1233>/
 <U0020><U0025><U0072><U0020><U0025><U005A><U0020><U0025><U0059>/
 <U0020><U130D><U002F><U120B>"
+week 7;19971130;1
 END LC_TIME
-%
diff -Nrup a/localedata/SUPPORTED b/localedata/SUPPORTED
--- a/localedata/SUPPORTED	2018-06-08 14:15:13.059877630 -0400
+++ b/localedata/SUPPORTED	2018-06-08 16:07:15.349302352 -0400
@@ -109,6 +109,7 @@ dv_MV/UTF-8 \
 dz_BT/UTF-8 \
 el_GR.UTF-8/UTF-8 \
 el_GR/ISO-8859-7 \
+el_GR@euro/ISO-8859-7 \
 el_CY.UTF-8/UTF-8 \
 el_CY/ISO-8859-7 \
 en_AG/UTF-8 \
